### PR TITLE
Stretch Support and Test Branch tests

### PR DESCRIPTION
**Pull request by cfcolaco**
Hi! 

This PR Merges to test branch changes made directly to master branch, which will make the full merge easier in the future and less prone to merging errors. 

- Added Support for Stretch
- Removed a few lines of code that appears to be useless (script worked just fine without them in all tests made). 

The following tests were made to the Test Branch reflecting already this commit:

**Debian VM 32Bit - TESTED -** Install Script OK as expected -  Not able to test VPN Connection
**Ubuntu 17.04 VM  - TESTED -** Install Script OK as expected - Not able to test VPN Connection
**Duvuan i386 VM . TESTED -** Install Script as expected - Not Able to test VPN Connection
**Duvuan x64 VM . TESTED -** Install Script as expected - Not Able to test VPN Connection

**AWS EC2 Debian 64Bit - TESTED -** Install Script as expected - Able to connect, VPN working as Expected.
**AWS EC2 UBUNTU 16.04 - TESTED -** Install Script as expected - Able to connect, VPN Working as Expected.
**raspbian Stretch - TESTED -** Install Script OK as Expected - Able to Connect, VPN working as Expected, able to reach devices inside Home Network. 

This solves issue #342 and probably #360  
I Think more issues got solved with this but will need to review them all latter!

### CODE REVIEW

**Comment by cfcolaco** [comment]
@redfast00 @0-kaladin take a look at this and tell me if there's anything that needs to be changed. =)

File: auto_install/install.sh [hunk 1]
```diff
@@ -61,14 +61,14 @@ dhcpcdFile=/etc/dhcpcd.conf
...
-Currently this installer supports Raspbian (or Devuan) jessie, Ubuntu 14.04 (trusty), and Ubuntu 16.04 (xenial).
+Currently this installer supports Raspbian, Debian or Devuan jessie and stretch and Ubuntu from 14.04 (trusty) to 17.04 (zetsy).
 If you think you received this message in error, you can post an issue on the GitHub at https://github.com/pivpn/pivpn/issues." ${r} ${c}
     exit 1
 }
 
 function maybeOS_Support() {
     if (whiptail --backtitle "Not Supported OS" --title "Not Supported OS" --yesno "You are on an OS that we have not tested but MAY work.
-Currently this installer supports Raspbian (or Devuan) jessie, Ubuntu 14.04 (trusty), and Ubuntu 16.04 (xenial).
+Currently this installer ssupports Raspbian, Debian or Devuan jessie and stretch and Ubuntu from 14.04 (trusty) to 17.04 (zetsy).
```
Comment (redfast00) [inline auto_install/install.sh]: Typo

Comment (cfcolaco) [reply auto_install/install.sh]: Fixed! =) sorry for that

**Comment by cfcolaco** [comment]
@redfast00 naaah, no worries, I apologize for that! forgo to review that text! =D

**Comment by redfast00** [comment]
Thank you!

**Comment by redfast00** [comment]
Do want to merge this into master?

**Comment by cfcolaco** [comment]
It's up to you and to @0-kaladin 
As I mentioned I tested it a lot in different distros and architectures and the test branch seems to be working on all of them! 

So as for me, The test branch is ok to be merged to Master! 
Maybe you and @0-kaladin want to make just a final test to make sure it's all fine .. don't know!

**Comment by Prunar** [comment]
Installed from test branch, worked fine :)

**Pull request merged**